stardog/templates/post-install-add-users.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "stardog.fullname" . }}-add-users
+  labels:
+    app.kubernetes.io/managed-by: {{.Release.Service | quote }}
+    app.kubernetes.io/instance: {{.Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    helm.sh/chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": post-upgrade,post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": before-hook-creation
+spec:
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/managed-by: {{.Release.Service | quote }}
+        app.kubernetes.io/instance: {{.Release.Name | quote }}
+        helm.sh/chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    spec:
+      restartPolicy: Never
+      imagePullSecrets:
+      {{- if .Values.image.existingPullSecret }}
+        - name: {{ .Values.image.existingPullSecret }}
+      {{- else }}
+        - name: {{ template "stardog.fullname" . }}-pullsecret
+      {{- end }}
+      containers:
+      - name: add-users
+        image: {{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+        - name: STARDOG_ADMIN_PW
+          valueFrom:
+            secretKeyRef:
+              name: {{ include "stardog.fullname" . }}-user-passwords
+              key: admin
+        {{- range .Values.stardog.users }}
+        - name: {{ .name }}_password
+          valueFrom:
+            secretKeyRef:
+              name: {{ include "stardog.fullname" $ }}-user-passwords
+              key: {{ .name }}
+        {{ end }}
+        command:
+        - /bin/sh
+        - -ce
+        - |
+          echo "testing stardog connectivity to http://{{ include "stardog.fullname" $ }}:5820"
+          until $(curl -o /dev/null -s -f {{ include "stardog.fullname" . }}); do sleep 1 && echo "."; done
+          {{- if .Release.IsInstall }}
+          echo Changing admin password
+          /opt/stardog/bin/stardog-admin --server http://{{ include "stardog.fullname" $ }}:5820 user passwd -N "$STARDOG_ADMIN_PW"
```

Comment:
Please use the full argument names to make it more readable.

Reply:
so instead of -N --new-password?

Reply:
yes